info/contributing.md

```diff
@@ -0,0 +1,16 @@
+![Contributing To Torrust Title Image](../img/2022_06_27-Contributing_To_TorrustTitle.png)
+
+# Contribution Guide for the Torrust Project
+
+__Thank You__ for your interest in our project, we always strive to make the onboarding process for new contributors as clear and comfortable as possible.
+
+To ensure that the project continues to improve without compromising its integrity and quality, we kindly ask that you follow this guide to help ensure that your contributions are harmonic as possible.
+
+## We have a Contributor Agreement
+Please take your time to review our [__contribution agreement__](./licensing/contributor_agreement_v01.md), all the work and effort that you contribute to this project will be governed by this document.
+
+## We have a Code of Conduct
+Please consider our [__Code of Conduct__](./code_of_conduct.md), this document helps you in the case that something goes wrong.
+
+_Note_, our contrubtor agreement is adapted from the standard [Contributor Covenant, v.1.4](https://www.contributor-covenant.org/version/1/4/code-of-conduct.html).
```

Comment:
our "contrubtor" agreement is adapted --- spelling mistake in ""
Change to:
 contributor